src/views/MainView/MainView.tsx

```diff
@@ -27,24 +26,24 @@ const MainView: React.FC = () => {
     const getClassName = () => {
         return classNames(
             'MainView', {
-                'InProgress': projectInProgress,
-                'Canceled': !projectInProgress && projectCanceled
-            }
+            'InProgress': projectInProgress,
+            'Canceled': !projectInProgress && projectCanceled
+        }
         );
     };
 
-    const DarkTooltip = withStyles(theme => ({
+    const DarkTooltip = styled(Tooltip)(({ theme }) => ({
```

Comment:
Something is wrong with that `Tooltip`. The background color is not applied.

Reply:
![image](https://user-images.githubusercontent.com/28984914/180602994-03069722-908c-4bd3-a7c9-5003ce168148.png)
I still see the background. can you please tell me where is it?

Reply:
ah I see,

Reply:
![image](https://user-images.githubusercontent.com/28984914/180603250-ff7a7fbb-d922-42ab-93a6-ace0a649f017.png)
Fixed